chapter_4_practic/task1.cpp

```diff
@@ -0,0 +1,32 @@
+﻿// A phone number, such as(212) 767 - 8900, can be thought of as having three parts : the
+// area code(212), the exchange(767), and the number(8900).Write a program that uses a
+// structure to store these three parts of a phone number separately.Call the structure
+// phone.Create two structure variables of type phone.Initialize one, and have the user
+// input a number for the other one.Then display both numbers.The interchange might
+// look like this :
+// Enter your area code, exchange, and number : 415 555 1212
+// My number is(212) 767 - 8900
+// Your number is(415) 555 - 1212
+
+#include <iostream>
+
+int main()
+{
+	struct Phone
+	{
+		int code;
```

Comment:
all structure members should have default initializers

Reply:
fixed in https://github.com/Dmitriy-Romanchuk/master/pull/4/commits/9d9d2ec85e205856ca38d8fcb47a519931b08150